esm.min.js

```diff
@@ -18,7 +18,7 @@ export const FormData = class FormData {
 constructor(...a){if(a.length)throw new TypeError(`Failed to construct 'FormData': parameter 1 is not of type 'HTMLFormElement'.`)}
 get [t]() {return 'FormData'}
 [i](){return this.entries()}
-[h](o) {return typeof o==='object'&&o[t]==='FormData'&&!m.some(m=>typeof o[m]!='function')}
+static [h](o) {return typeof o==='object'&&o[t]==='FormData'&&!m.some(m=>typeof o[m]!='function')}
```

Comment:
```js
typeof object === "object" && object[Symbol.toStringTag] === "FormData"
```

This assertion will cause an error when `object` is `null`, same issues as https://github.com/node-fetch/fetch-blob/issues/58
I also think that you'll better check at least some methods existence. I think these assertions would be more than enough: https://github.com/octet-stream/form-data-encoder/blob/a94e5e4c37304334349b902d094048e380ff6ab2/lib/util/isFormData.ts#L10-L18

Reply:
Thx, fixed